doc/DeveloperGuide.md

```diff
@@ -87,6 +87,35 @@ Use case ends.
 
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
+  
+#### Use case: Rename tag
+  
+**MSS**
+
+1. User input to change tag
+2. AddressBook request user to input tag to change
+3. User enters the tag to change
+4. AddressBook shows list of person with that tag
+5. User enters the new tag to change to
+6. AddressBook request for confirmation to change tag for all person with the initial tag
+7. User confirms the change
+8. AddressBook rename all tags to the new tag
```

Comment:
Keep in mind that (especially for the project) that the app should have `advanced CLI`.